docs/getting-started.md

```diff
@@ -0,0 +1,168 @@
+# Getting started
+
+When running this simulator there are two main components: the simulator and the ros bridge.
+Your autonomous system and the ros bridge should use the same ros core, either by running on the same computer or through a network connection.
+
+This page is an overview of the different methods to get these components up and running.
+
+## Running the simulator
+
+To run the simulation smoothly you need quite a fast windows computer with a modern videocard.
+We highly recommend the following computer specs. You might be able to run with less power but everything will be slower.
+For developing this project, you need quite a good computer because Unreal Engine is a heavy baby.
+
+* 8 core 2.3Ghz CPU
+* 12 GB memory
+* 30GB free SSD storage (120GB when building the unreal project from source)
+* Recent NVidia card with Vulkan support and 3 GB of memory. (You can check the video card drivers by running `vulkaninfo`)
+
+If your computer does not suffice you can use a remote workstation on Google Cloud Platform.
+Read [this tutorial](gcp-remote-workstation.md) on how to setup your virtual workstation.
+
+The simulator will load settings from the file `Formula-Student-Driverless-Simulator/settings.json` in your **home directory**.
+This file contains the sensor configuration of the car.
+Copy-paste the contents of the [settings.json file at the root of this repository](https://github.com/FS-Online/Formula-Student-Driverless-Simulator/blob/master/settings.json) inside.
+This should get you started with the default sensor configuration, feel free to try your own custom sensor suite.
```

Comment:
This part confuses me. Currently I'm running on an Ubuntu instance and I just have a settings.json file inside the clone of this repository, so clearly these steps do not apply in all cases. We should make clear if the settings.json file has to be in the home directory or the Formula-Student-Driverless-Simulator folder. Adding something like, unless you have cloned this repository in the home directory, copy-paste the contents...

Reply:
Clarified it, how do you feel about it now?

Reply:
great!